cli/index.md

```diff
@@ -310,12 +310,12 @@ duck --username kahy9boj3eix --upload azure://kahy9boj3eix.blob.core.windows.net
 
 ### Download Files Matching Glob Pattern from FTP
 ```{code-block}
-duck -v --download ftp://mirror.switch.ch/mirror/apache/dist/httpd/*.gz ~/Downloads
+duck --user anonymous --verbose --download s3:/profiles.cyberduck.io/Wasabi* ~/Downloads/
```

Comment:
Is there a specific reason why FTP was changed to S3?

Reply:
Please align the header "Download Files Matching Glob Pattern from FTP" to the displayed content within the code block.